### PR TITLE
Document Basic Auth constraints on Linux and macOs

### PR DESCRIPTION
**Pull request by dantraMSFT**
Document the change to PowerShell disabling support for Basic Authentication over HTTP and document alternatives.

Fix https://github.com/PowerShell/PowerShell-Docs/issues/2394

Version(s) of document impacted
------------------------------
- [X] Impacts 6.1 document
- [ ] Impacts 6.0 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [X] This issue only shows up in version (6.1) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by dantraMSFT** [comment]
NOTE: See https://github.com/PowerShell/psl-omi-provider/issues/118 for Basic Auth over HTTPS

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md](https://github.com/dantraMSFT/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md) | :warning:Warning | | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Remote_Requirements.md)

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Remote_Requirements.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md](https://github.com/dantraMSFT/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C5%5C2%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201805022220005488-2395%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=THr81LI%2BOS6yWya6BIVCZNK0PC1UGUZFA3Bc3xOXNE0%3D&st=2018-05-02T22%3A24%3A26Z&se=2018-06-02T22%3A29%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md](https://github.com/dantraMSFT/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md) | :warning:Warning | | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Remote_Requirements.md)

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Remote_Requirements.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md](https://github.com/dantraMSFT/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C5%5C2%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201805022343097767-2395%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=gxpf8AtuhNLrG1%2FzXh6%2F5hYMZ6UURz9XGIHMWTdYIHc%3D&st=2018-05-02T23%3A47%3A22Z&se=2018-06-02T23%3A52%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md](https://github.com/dantraMSFT/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md) | :warning:Warning | | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Remote_Requirements.md)

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Remote_Requirements.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md](https://github.com/dantraMSFT/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C5%5C3%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201805031659279431-2395%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=4EXC7WfRmXEqbH5aSFyL4SL7TgdUEp0sluIFyjtOqsw%3D&st=2018-05-03T17%3A04%3A19Z&se=2018-06-03T17%3A09%3A19Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md](https://github.com/dantraMSFT/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md) | :warning:Warning | | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Remote_Requirements.md)

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Remote_Requirements.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md](https://github.com/dantraMSFT/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C5%5C3%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201805031808350302-2395%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=5wIWk0l4TqiCNuASMIXamJSa9o0kyFcHhxTpyBhnXlU%3D&st=2018-05-03T18%3A13%3A17Z&se=2018-06-03T18%3A18%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md [hunk 1]
```diff
@@ -225,6 +225,55 @@ In general, before you connect and as you are establishing the connection, the
 policies on the local computer are in effect. When you are using the
 connection, the policies on the remote computer are in effect.
 
+## Basic Authentication Limitations on Linux and macOs
```
Comment (SteveL-MSFT) [inline reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md]: `macOS` - throughout doc

File: reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md [hunk 2]
```diff
@@ -225,6 +225,55 @@ In general, before you connect and as you are establishing the connection, the
...
+over HTTP is not supported. Basic Authentication can be used over HTTPS by
+installing a certificate on the target server. The certificate must have a
+CN name that matches the hostname, is not expired or revoked. A self-signed
+certificate may be used for testing purposes.
+
+See [How To: Configure WINRM for HTTPS](https://support.microsoft.com/en-us/help/2019527/how-to-configure-winrm-for-https)
+for additonal details.
+
+The following command, run from an elevated command prompt, will configure the
+HTTPS listener with the installed certificate.
```
Comment (SteveL-MSFT) [inline reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md]: I think you should change to "configure the HTTPS listener on Windows..."

File: reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md [hunk 3]
```diff
@@ -225,6 +225,55 @@ In general, before you connect and as you are establishing the connection, the
...
+
+```powershell
+$hostinfo = '@{Hostname="<DNS_NAME>"; CertificateThumbprint="<THUMBPRINT>"}'
+winrm create winrm/config/Listener?Address=*+Transport=HTTPS $hostinfo
+```
+
+On the Linux or macOs side, select Basic for authentication and -UseSSl.
+
+```powershell
+# The specified local user must have administrator rights on the target machine.
```
Comment (SteveL-MSFT) [inline reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md]: Perhaps be explicit that Basic can't be used with domain accounts

**Comment by JumpingYang001** [comment]
want to reminder that both libmi.dylib, libpsrpclient.dylib, libmi.so and libpsrpclient.so need to pack into PoweShell if user want to use new release like https://github.com/PowerShell/psl-omi-provider/releases/tag/v1.4.2-2 , thanks.

File: reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md (hunk 1 above)
Comment (dantraMSFT) [reply reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md]: Fixed

File: reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md (hunk 2 above)
Comment (dantraMSFT) [reply reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md]: Fixed

File: reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md (hunk 3 above)
Comment (dantraMSFT) [reply reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md]: Fixed

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md](https://github.com/dantraMSFT/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md) | :warning:Warning | | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Remote_Requirements.md)

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Remote_Requirements.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md](https://github.com/dantraMSFT/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Remote_Requirements.md)
 - **[Warning]** Ignore metadata: locale. They are generated by open publish.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C5%5C8%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201805081615318297-2395%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=w4OZ5L%2BXqc1SwhWpYJxtdEfxzoByNms001cJruATBqw%3D&st=2018-05-08T16%3A20%3A18Z&se=2018-06-08T16%3A25%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by dantraMSFT** [comment]
@JumpingYang001 - I'm not sure what you are asking for.  I've already checked in the change to PowerShell to reference 1.4.2-2.

**Pull request merged**